tests/integration/components/qrcode-test.js

```diff
@@ -0,0 +1,16 @@
+import { module, test } from 'qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { setupRenderingTest } from 'ember-qunit';
+
+module('Integration | Component | qr-code', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('QR code component and text render on mobile route', async function (assert) {
+    this.set('userId', '12345');
+    await render(hbs`
+     <Qrcode @text = {{this.userId}} />
```

Comment:
We are testing here the component i.e. `qrcode.hbs` but what is the point of testing this component when we aren't using it ?
For reference, please check this comment

Reply:
fixed